documentation/api/v1/loot_api_doc.rb

```diff
@@ -10,14 +10,18 @@ module LootApiDoc
   LTYPE_EXAMPLE = "'file', 'image', 'config_file', etc."
   PATH_DESC = 'The on-disk path to the loot file.'
   PATH_EXAMPLE = '/path/to/file.txt'
-  DATA_DESC = 'The contents of the file.'
+  DATA_DESC = "Base64 encoded copy of the file's contents."
+  DATA_EXAMPLE = 'dGhpcyBpcyB0aGUgZmlsZSdzIGNvbnRlbnRz'
   CONTENT_TYPE_DESC = 'The mime/content type of the file at {#path}.  Used to server the file correctly so browsers understand whether to render or download the file.'
   CONTENT_TYPE_EXAMPLE = 'text/plain'
   NAME_DESC = 'The name of the loot.'
   NAME_EXAMPLE = 'password_file.txt'
   INFO_DESC = 'Information about the loot.'
   MODULE_RUN_ID_DESC = 'The ID of the module run record this loot is associated with.'
 
+  # Some of the attributes expect different data when doing a create.
+  CREATE_PATH_DESC = 'The name to give the file on the server. Note that this will prepend a unique string to the given value.'
```

Comment:
Minor: I found the following a little confusing, consider rewording. "Note that this will prepend a unique string to the given value." Suggestion: "A unique string will be prepended to the filename portion of the full path."

Reply:
I wanted to somehow note that when you're creating a loot object, the full path isn't actually honored. We save everything in the `~/.msf4/loot` directory when the Base64 is processed. If you have a full file path in there, we just chop off everything but the filename.

Your suggested edit makes it seem like we'll still honor the full path value that the user specifies, and just prepend the unique string after the last `/` but before the actual file name. I'll use a version of that suggestion that makes it more obvious only the file name is needed on a create operation.